### PR TITLE
Support reading Postgres array as JSON

### PR DESCRIPTION
**Pull request by kasiafi**
Fixes: #682

### CODE REVIEW

**Comment by kasiafi** [comment]
Issue:  https://github.com/prestosql/presto/issues/682

**Comment by kasiafi** [comment]
cc @guyco33

File: presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcConnector.java [hunk 1]
```diff
@@ -25,16 +25,19 @@
 import io.prestosql.spi.connector.ConnectorSplitManager;
```
Comment (kokosing) [inline presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcConnector.java]: Can you please extract this commit as separate PR so we could merge this without blocking on postgres changes?

Comment (kasiafi) [reply presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcConnector.java]: Done: https://github.com/prestosql/presto/pull/1181

**Comment by kasiafi** [comment]
Applied comments

**Comment by findepi** [comment]
Merged, thanks!
